sdk/tests/deqp/framework/opengl/simplereference/sglrReferenceContext.js

```diff
@@ -3257,12 +3258,16 @@ goog.scope(function() {
                 access = colorBuf.getSubregion(colorArea);
 
                 if (!maskUsed)
-                    access.clear(value);
+                    access.clear(isSRGB ? tcuTextureUtil.linearToSRGB(value) : value);
```

Comment:
Thanks for review.  I updated this pr to align with C++ code.
https://android.googlesource.com/platform/external/deqp/+/deqp-dev/framework/opengl/simplereference/sglrReferenceContext.cpp#3466